docs/templates/override/styles/docfx.vendor.js

```diff
@@ -0,0 +1,38 @@
+// @license-end
```

Comment:
This is a straight copy of the `docfx.vendor.js` file from the default template, with `highlight.js` taken out so I could swap it out for another version (fixing an important bug). More context: https://github.com/dotnet/docfx/issues/7373